iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/SubnetworkImpl.java

```diff
@@ -232,6 +232,11 @@ public LineAdder newLine() {
         return getNetwork().newLine(id);
     }
 
+    @Override
+    public LineAdder newLine(Line line) {
+        return getNetwork().newLine(line.getId());
```

Comment:
I think that it is more :
```suggestion
        return getNetwork().newLine(id, line);
```
(using the method newLine(String subnetwork, Line line) )
What you wrote here is creating a new (empty) line adder in the subnetwork that is named line.getId() which is not what we want